CoEpi/storage/DBCENKey.swift

```diff
@@ -0,0 +1,23 @@
+import RealmSwift
+
+final class DBCENKey : Object {
+    @objc dynamic var timestamp: Int = Int(Date().timeIntervalSince1970)
+    @objc dynamic var CENKey: String = ""
+    var latitude: Double? = nil
+    var longitude: Double? = nil
+    
+    override static func primaryKey() -> String? {
+        return "timestamp"
+    }
+    
+    required init( _ts: Int, _cenKey: String ) {
+        self.timestamp = _ts
+        self.CENKey = _cenKey
+    }
+    
+    required init() {
```

Comment:
Do we need this?

Reply:
Yes ... though not entirely sure why.  XCode complains if `required init()` not defined